plugins/SitesManager/vue/src/ManageGlobalSettings/ManageGlobalSettings.vue

```diff
@@ -251,23 +232,23 @@ interface GlobalSettingsState {
   searchKeywordParametersGlobal: string[];
   searchCategoryParametersGlobal: string[];
   isSaving: boolean;
+  exclusionTypeForQueryParams: string;
 }
 
 interface IpFromHeaderResponse {
   value: string;
 }
 
 export default defineComponent({
-  props: {
-    // TypeScript can't add state types if there are no properties (probably a bug in Vue)
-    // so we add one dummy property to get the compile to work
-    dummy: String,
-  },
   components: {
+    ExcludeQueryParameterSettings,
     ContentBlock,
     Field,
     SaveButton,
   },
+  props: {
+    commonSensitiveQueryParams: [],
```

Comment:
```suggestion
    commonSensitiveQueryParams: {
      type: Array as PropType<string[]>,
      default: () => [],
    },
```

Otherwise the Vue internals are not happy:

> [Vue warn]: Invalid prop: type check failed for prop "commonSensitiveQueryParams". Expected , got Array